docs/guide-es/caching-data.md

```diff
@@ -9,7 +9,7 @@ El siguiente código muestra el típico patrón de uso para el almacenamiento en
 // intenta recuperar $data de la caché
 $data = $cache->get($key);
 
-if ($data === false) {
+if ($data === `false`) {
```

Comment:
should not be done inside of PHP code examples. I'm afraid the PR is useless this way :)